### PR TITLE
Br 1874 annotations

### PR DESCRIPTION
**Pull request by danbowles**

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -102,6 +105,13 @@ export function ajaxWidgetData(widgetId, fetchUrl, headersAttr = null) {
         return;
       }
 
+      try {
+        apiData.annotations = JSON.parse(json.data.annotations);
+      } catch (err) {
+        console.log(err.message); // eslint-disable-line no-console
```
Comment (mviggiano) [inline app/actions/index.js]: Maybe a `console.error` or `console.warn` here instead?

File: app/components/ChartAnnotations/index.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    const makeAnnotations = d3v4.annotation()
+      .type(d3v4.annotationCalloutElbow)
+      .annotations(annotationsWithCoords)
+      .editMode(editing)
+      .on('dragend', ChartAnnotations.onAnnotationDragEnd);
+
+    containerEl.append('g')
+      .attr('class', 'annotations-group')
+      .call(makeAnnotations);
+    console.log('rendered annos');
```
Comment (mviggiano) [inline app/components/ChartAnnotations/index.js]: Still need to be here?

File: app/components/ChartSettings/modules/Annotations.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+// import { Button, Panel, PanelHeader, PanelFooter, Switch } from 'rebass';
```
Comment (mviggiano) [inline app/components/ChartSettings/modules/Annotations.js]: Remove?

File: app/actions/index.js (hunk 1 above)
Comment (danbowles) [reply app/actions/index.js]: done

File: app/components/ChartAnnotations/index.js (hunk 2 above)
Comment (danbowles) [reply app/components/ChartAnnotations/index.js]: nope. removed :)

**Pull request merged**